docs/pages/admin-guides/access-controls/access-request-plugins/ssh-approval-datadog.mdx

```diff
@@ -0,0 +1,160 @@
+to view your organization's API keys. To generate a new API key, click on **New Key**
+and name it "Teleport Access". Copy the API key to paste in a later step.
+
+![Datadog dashboard for API Keys](../../../../img/enterprise/plugins/datadog/dashboard-api-keys.png)
+
+### Create a Datadog Service Account
+
+Next, navigate to **Organization Settings -> Identity & Accounts -> Service Accounts**
+to view your organization's service accounts. To create a new service account, click
+on **New Service Account** and name it "Teleport Access". You can provide whatever
+email you'd like to associate with the service account, then assign the role
+"Datadog Standard Role". This role provides the required permissions necessary to
+manage the Teleport Access Request incidents.
+
+![Datadog dashboard for Service Accounts](../../../../img/enterprise/plugins/datadog/dashboard-service-accounts.png)
+
+Now click on the "Teleport Access" service account that was just created and click
+on the **Application Keys** tab and click on **New Key** and name it "Teleport Access"
+to create a new Application key. Copy the Application key to paste in a later step.
+
+![Datadog dashboard for Application Keys](../../../../img/enterprise/plugins/datadog/dashboard-application-keys.png)
+
+## Step 4/5. Enroll the Datadog Incident Management plugin
+
+At this point, you're now ready to enroll the Datadog Incident Management plugin.
+Navigate to **Access Management -> Enroll New Integration -> Datadog**.
+
+![Select enrollment](../../../../img/enterprise/plugins/datadog/select-enrollment.png)
+
+Provide the API and Application keys generated above. Select the desired API endpoint.
+Then provide the Datadog team handle, that you created earlier, as the fallback recipient.
+This should be "teleport-access". The fallback recipient will be the default recipient
+for notifications. You can configure more custom notification routing rules afterwards
+using Access Monitoring Rules.
+
+![Datadog enrollment](../../../../img/enterprise/plugins/datadog/datadog-enrollment.png)
+
+If the recipient is a Datadog team, the team name will be added to the Datadog incident
+teams attribute. The plugin will not manually send notifications to members of the
+team. Instead you will want to navigate to **Incidents -> Settings -> Notifications -> Rules**
+and add a notification rule for when an incident with the matching teams attribute
+is created or updated.
+
+![Datadog dashboard for Incident Notification Rules](../../../../img/enterprise/plugins/datadog/dashboard-notification-rules.png)
+
+<Details title="Suggested reviewers">
+
+Users can suggest reviewers when they create an Access Request, e.g.,:
+
+```code
+$ tsh request create --roles=dbadmin --reviewers=alice@example.com,ivan@example.com
+```
+
+If an Access Request includes suggested reviewers, the Datadog Incident Management
+plugin will add these to the list of users to notify.
```

Comment:
What's the format of sugested reviewrs? emails? do we also support other team names? Can datadog users have usernames that are not emails?